rimage/file_simple.c

```diff
@@ -72,6 +73,73 @@ static int get_mem_zone_type(struct image *image, Elf32_Shdr *section)
 	return SOF_FW_BLK_TYPE_INVALID;
 }
 
+static int fw_version_copy(struct snd_sof_logs_header *header,
```

Comment:
I feel like the separating out of this function can be its own patch.

Reply:
If it were just refactor, then yea. But here you'd have 1 diff to have it changed in another.